<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_order</class>
 <widget class="QWidget" name="add_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1414</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#add_order{
background: rgb(240, 237, 255);
border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="8" column="0" colspan="18">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2 {
background: rgb(46, 54, 58);
border: none;
border-right: 1px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>153</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>152</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="17">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="9">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" colspan="16">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>280</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#treeWidget {
    border: 3px solid rgb(123, 97, 170);
    border-radius: 3px;
}</string>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox_vendor">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Fournisseur :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Num√©ro :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_montant">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="13">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Hors articles :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="14">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Montant TTC :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="12">
    <widget class="QPushButton" name="pushButton_delImg">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10" colspan="3">
    <widget class="QLineEdit" name="lineEdit_number">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="11">
    <widget class="QLineEdit" name="lineEdit_path">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="14">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Date :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="18">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame {
background: rgb(46, 54, 58);
border: none;
border-right: 1px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Ajouter une commande</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="suffix">
      <string> ‚Ç¨</string>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QPushButton" name="pushButton_addImg">
     <property name="text">
      <string>üìé Joindre une facture</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="15">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item row="3" column="16">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Auj.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="15">
    <widget class="QPushButton" name="pushButton_addArticle">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Nouvel article</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_vendor</tabstop>
  <tabstop>lineEdit_number</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>pushButton_addImg</tabstop>
  <tabstop>lineEdit_path</tabstop>
  <tabstop>pushButton_delImg</tabstop>
  <tabstop>pushButton_add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
